tests/cpp_api/audio_tests/audio_tests.cpp

```diff
@@ -143,16 +186,15 @@ int test(int test_case, const char *path, int qa_mode, int downmix, int gpu) {
         return -1;
     }
 
-    std::string file_list_path;  // User can modify this with the file list path if required
-    if (qa_mode) {                    // setting the default file list path from ROCAL_DATA_PATH
+    std::string file_list_path = "";  // User can modify this with the file list path if required
```

Comment:
why are we brining back the `""` initialization

Reply:
Removed "" initialization